server/main.js

```diff
@@ -32,6 +34,29 @@ app.route('/candname')
       })
   })
 
+  //http://localhost:8080/candtwitter?candtwitternickname=RepByrne
+app.route('/candtwitter')
+.get(function(req,res){
+  var candidatenickname =req.query.candtwitternickname;
+    twit.gettweets(candidatenickname).then(function(results){
+      res.status(200).send(results);
+    }).catch(function(error){
+      console.log(error);
+       var obj = {};
+                    obj.created_at = '';
+                    obj.text = '';
+                    obj.user = '';
+                    obj.location = '';
+                    obj.followers = '';
+                    obj.url = '';
+
+                    var array=[];
+                    array.push(obj);
+                  res.status(200).send(array);
```

Comment:
allows us to populate the field even with no response (from the server side). 

We can discuss a refactor during code cleanup week.